rust/rope/src/tree.rs

```diff
@@ -478,7 +471,29 @@ impl<N: NodeInfo> TreeBuilder<N> {
     pub fn push(&mut self, n: Node<N>) {
         match self.0.take() {
             None => self.0 = Some(n),
-            Some(buf) => self.0 = Some(Node::concat(buf, n))
+            Some(buf) => self.0 = Some(Node::concat(buf, n)),
+        }
+    }
+
+    pub fn push_leaves(&mut self, leaves: Vec<N::L>) {
```

Comment:
Being a pub fn, this should have at least a short docstring. I know that's not upheld in the existing code, but trying to raise the standard...